StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -163,7 +163,7 @@ extension PaymentSheetError: CustomDebugStringConvertible {
             case .linkSignUpNotRequired:
                 return "Don't call sign up if not needed"
             case .noPaymentMethodTypesAvailable(intentPaymentMethods: let intentPaymentMethods):
-                return "None of the payment methods on the PaymentIntent/SetupIntent can be used in PaymentSheet: \(intentPaymentMethods). You may need to set `allowsDelayedPaymentMethods` or `allowsPaymentMethodsRequiringShippingAddress` in your PaymentSheet.Configuration object."
+                return "None of the payment methods on the PaymentIntent/SetupIntent can be used in PaymentSheet: \(intentPaymentMethods). You may need to set `allowsDelayedPaymentMethods` or `allowsPaymentMethodsRequiringShippingAddress` or set `returnURL` in your PaymentSheet.Configuration object."
```

Comment:
It'd be even better if we could say the specific reason to this error message but this is a perfectly fine incremental improvement.

Reply:
+1 I think the path of the code that is calling this makes it difficult to know the exact reason. Looking for an incremental improvement at this point.